app/assets/stylesheets/scaffolds.scss

```diff
@@ -0,0 +1,73 @@
+body {
```

Comment:
@NoorAlkattan GREAT! Now, you will need to include this stylesheet in the application.css file so it becomes visible to the code, and you will need to start decorating your html.erb files using the custom styles.